warcprox/dedup.py

```diff
@@ -174,3 +186,77 @@ def notify(self, recorded_url, records):
             else:
                 self.save(digest_key, records[0])
 
+
+def _split_timestamp(timestamp):
+    """split `timestamp` into a tuple of 6 integers.
```

Comment:
Why not use `strptime`?

Reply:
I used this because it is what we did in PWM and it was faster. I've changed it to ``strptime``. NP.